src/chrome/content/rules/Yeti-designs.com.xml

```diff
@@ -2,5 +2,10 @@
 	<target host="yeti-designs.com" />
 	<target host="www.yeti-designs.com" />
 
+<!--	Redirect to www (connection is refused without it):
```

Comment:
Sorry I wasn't clear. Could you add this in the top comment please? 

Something like

```
<!--
	Refused:
		yeti-designs.com

-->
```

(or whatever format you prefer)